dotnet-sdk-90/snap/snapcraft.yaml

```diff
@@ -0,0 +1,100 @@
+name: dotnet-sdk-90
+base: core22
+adopt-info: dotnet-sdk
+summary: .NET 9 SDK
+description: |
+  The .NET SDK is a set of libraries and tools that developers use to create
+  .NET applications and libraries.
+
+grade: stable
+confinement: strict
+
+package-repositories:
+ - type: apt
+   ppa: dotnet/snaps
+   priority: always
+
+architectures:
+  - build-on: [amd64]
+    build-for: [amd64]
+  - build-on: [arm64]
+    build-for: [arm64]
+
+slots:
+  dotnet-sdk:
+    interface: content
+    content: dotnet-sdk-90
+    source:
+      read:
+        - $SNAP/usr/lib/dotnet
+
+parts:
+  dotnet-sdk:
+    plugin: nil
+    source: .
+    stage-packages:
+      - dotnet-sdk-9.0
+      - dotnet-sdk-aot-9.0
```

Comment:
**note:** `dotnet9` should result in the same

Reply:
Yes, this was on purpose. I wanted the `build-packages` entry to be as explicit as possible conveying that this snap also includes the NativeAOT binaries along with the SDK, which is not the case for past version, which only include the SDK binaries.